<?xml version="1.0"?>
<launch>
  <arg name="namespace" default="wamv"/>
  <arg name="linear_scaling" default="0.1" />
  <arg name="angular_scaling" default="0.05" />

  
  <node pkg="rbot280" type="test_pid.py"
	name="diff_drive_control" output="screen">
    <!-- set inital pid gains -->
    <!-- velocity twist -->
    <param name='velKp' value='0.1'/>
    <param name='velKi' value='0.0'/>
    <param name='velKd' value='0.0'/>
    <!-- yaw torque pid gains -->
    <param name='yawKp' value='1.0'/>
    <param name='yawKi' value='0.001'/>
    <param name='yawKd' value='0.0'/>

    <!-- remap to make more generic -->
    <remap to="/$(arg namespace)/thrusters/left_thrust_cmd" from="left_thrust_cmd"/>
    <remap to="/$(arg namespace)/thrusters/right_thrust_cmd" from="right_thrust_cmd"/>
  </node>
  

</launch>
